### PR TITLE
Updated wording for query buffer management

### PR DESCRIPTION
**Pull request by macintux**
Per Andrei's clarification on #2410, a new PR to clarify the role the manager plays.

### CODE REVIEW

File: content/riak/ts/1.6.0/configuring/riakconf.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ riak_kv.query.timeseries.max_returned_data_size = 10*1000*1000
 
 `riak_kv.query.timeseries.qbuf_inmem_max`: Max heap size of the query buffer manager process before query buffers start to be written out to the leveldb backend.  Default value is "10MB".
 
-The query buffer manager tries to keep collected query results in memory for faster retrieval.  To avoid accumulating excessive amounts of data, this parameter sets the max process heap limit the query buffer manager can allocate before it dumps the entire query buffer with which the limit was hit, to the leveldb backend (and proceeds to add eventual chunks to it).
+For any `LIMIT` or `ORDER BY` timeseries queries, a query buffer manager is created. It initially stores query results in memory for faster retrieval.  To avoid accumulating excessive amounts of data, this parameter sets the max process heap limit the query buffer manager can allocate before it dumps the entire query buffer with which the limit was hit, to the leveldb backend for further write and read activity.
```
Comment (hmmr) [inline content/riak/ts/1.6.0/configuring/riakconf.md]: Can you refactor the comment per https://github.com/basho/basho_docs/pull/2410/files#r98118239? Essentially, all I contend with is the suggestion of many query buffer _managers_, where there is only one that I read in your edits. The minimal change to get things straight, would be to omit "manager", thus: "For any ..., a query buffer is created."

Comment (macintux) [inline content/riak/ts/1.6.0/configuring/riakconf.md]: Refactored.

**Comment by hmmr** [comment]
+1

**Pull request merged**